src/components/project/project.rules.ts

```diff
@@ -85,6 +87,11 @@ export class ProjectRules {
   ): Promise<StepRule> {
     const mostRecentPreviousStep = (steps: ProjectStep[]) =>
       this.getMostRecentPreviousStep(id, steps, changeset);
+    const financialApprovers = async () => [
+      ...(await this.financialApproverRepo.read(projectType)).map(
+        ({ user }) => user.id,
+      ),
+    ];
```

Comment:
There's no need to spread the array into another array.
```suggestion
    const financialApprovers = async () =>
      (await this.financialApproverRepo.read(projectType)).map(
        ({ user }) => user.id,
      );
```

Reply:
If I don't spread it, then I get errors elsewhere, 
```
[{
	"resource": "/Users/andreturner/Dev/Repos/cord-api-v3/src/components/project/project.rules.ts",
	"owner": "typescript",
	"code": "2322",
	"severity": 8,
	"message": "Type '() => Promise<(string | ID[])[]>' is not assignable to type 'Notifiers | undefined'.\n  Type '() => Promise<(string | ID[])[]>' is not assignable to type '() => Promisable<Many<Notifier>>'.\n    Type 'Promise<(string | ID[])[]>' is not assignable to type 'Promisable<Many<Notifier>>'.",
	"source": "ts",
	"startLineNumber": 508,
	"startColumn": 11,
	"endLineNumber": 508,
	"endColumn": 23,
	"relatedInformation": [
		{
			"startLineNumber": 49,
			"startColumn": 3,
			"endLineNumber": 49,
			"endColumn": 15,
			"message": "The expected type comes from property 'getNotifiers' which is declared here on type 'StepRule'",
			"resource": "/Users/andreturner/Dev/Repos/cord-api-v3/src/components/project/project.rules.ts"
		}
	]
}]
```

Reply:
That must be VSCode lying to you

Reply:
okie dokie, I can push it without, but not sure how the 6 errors are showing up for me but not for you.

Reply:
Remove the array brackets too

Reply:
ok, thank you.